### DISCUSSION TITLE
[@types/d3-org-chart] missing `getChartState` function definition

### DISCUSSION THREAD

**Discussion by marcellino-ornelas** [DefinitelyTyped/DefinitelyTyped#60502] [Issues with a @types package]
[@types/d3-org-chart] missing `getChartState` function definition 😃 

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60501

[a comment by an automated account is not shown]

**Comment by domdomegg** [comment]
Thanks for raising this! I've raised a PR that I think should address this:

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60503

If you have time, I'd appreciate if you could look over it and review it. Feel free to add a test case specific to what you're using it for (I've used the example from the linked GitHub issue https://github.com/bumbeishvili/org-chart/issues/136#issuecomment-1080349067)
